pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>gov.cms.madie</groupId>
   <artifactId>madie-java-models</artifactId>
-  <version>0.2.25-SNAPSHOT</version>
+  <version>0.2.26-SNAPSHOT</version>
```

Comment:
looks like you missed pushing actual change.

Reply:
That would seem so